Header header
string uav_id
float64 start_time
float64 duration
uint32 num_segments
float64[] segment_times
float64[] px_coeffs  # x方向多项式系数
float64[] py_coeffs  # y方向多项式系数
float64[] pz_coeffs  # z方向多项式系数
